examples/omega2/expansion_board.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
```

Comment:
This will not work on a pure $5 Omega2. The GRB LED is located on the expansion board